gsc/_t-univ-3.scm

```diff
@@ -2104,7 +2104,9 @@
                (^float-floor expr)))
 
     ((ruby)
-     (^ expr ".truncate"))
+     (^if-expr (^ "[0, Float::INFINITY, -Float::INFINITY, Float::NAN].include?(" expr ")")
+               expr
+               (^call-prim (^member expr 'truncate))))
```

Comment:
This test is too expensive... better would be:
```
  (^if-expr (^float-finite? expr) (^call-prim (^member expr 'truncate)) expr)
```

But for this to work, the function ```univ-emit-float-finite?``` needs to be augmented with the case:
```
  ((ruby)
   (^call-prim (^member expr 'finite?)))
```